pulsar-broker/src/test/java/org/apache/pulsar/broker/admin/CreateSubscriptionTest.java

```diff
@@ -135,6 +135,46 @@ public void createSubscriptionOnPartitionedTopic() throws Exception {
         }
     }
 
+    public void testContainsInvisibleCharactersForSubscriptionName() throws PulsarAdminException {
+        String topic = "persistent://my-property/my-ns/my-partitioned-topic";
+        admin.topics().createPartitionedTopic(topic, 10);
+        admin.topics().createSubscription(topic, "sub-1", MessageId.latest);
+        admin.topics().createSubscription(topic, "sub~`!@#$￥%^……&*(){「【[]}】』\\|、:;\"'<《,>。》.?/-1",
+                MessageId.latest);
+
+        // Create should fail if the subscription name contain invisible characters
+        try {
+            admin.topics().createSubscription(topic, "sub\u0000-2", MessageId.latest);
+            fail("Should have failed");
+        } catch (Exception e) {
+            // Expected
```

Comment:
could we check the response status and error message?

Reply:
Fixed! @poorbarcode